DATA/production/configurations/2022/LHC22f/apass1/getTPCvdrift.C

```diff
@@ -0,0 +1,63 @@
+#include <cmath>
+#include <fmt/format.h>
+#include <string_view>
+#include <fstream>
+
+#include "TSystem.h"
+
+#include "CCDB/CcdbApi.h"
+#include "DataFormatsTPC/LtrCalibData.h"
+#include "TPCBase/ParameterGas.h"
+
+float getTPCvdrift(int run, std::string_view ltrUrl = "http://ccdb-test.cern.ch:8080")
```

Comment:
@chiarazampolli , please note that Robert changed the ccdb-url for the laser calibration to `alice-ccdb` from run 518737 on. So for the 13TeV data you either need to pass this as an additional parameter to the `getTPCvdrift` script, or change the default value here:
```suggestion
float getTPCvdrift(int run, std::string_view ltrUrl = "http://alice-ccdb.cern.ch")
```

Reply:
Ok, good to know. Then we need to change also for 22e, and reprocess those runs. 

Chiara